packages/ui/cypress/component/STooltip.spec.cy.ts

```diff
@@ -0,0 +1,116 @@
+import { mount } from '@cypress/vue'
+import { STooltip } from '@/lib'
+
+const testIdSelector = (id: string) => `[data-testid=${id}]`
```

Comment:
Isn't it defined in some utils module already?

Reply:
As far as I know, no